relay-general/src/protocol/stacktrace.rs

```diff
@@ -93,6 +95,7 @@ pub struct Frame {
     pub pre_context: Annotated<Array<String>>,
 
     /// Source code of the current line (`lineno`).
+    #[metastructure(skip_serialization = "empty")]
```

Comment:
This should be `null`. We use `context_line` in Sentry to check if there is source context, and in theory the context line can be an empty string.